hypervisor/lxc/lxc.go

```diff
@@ -43,6 +69,114 @@ type LXCHypervisorDriver struct {
 	name      string
 }
 
+func modifyConf() {
+	f, err := ioutil.ReadFile(LxcConfigFile)
+
+	if err != nil {
+		log.Fatalf("Failed loading lxc default.conf: ", err)
+	}
+
+	cf := cleanConfigFile(string(f))
+
+	var newSettings string
+	//for i, _ := range interfaces {
+	newSettings = updateSettings(interfaces[0], newSettings)
+	//}
+
+	result := strings.Join([]string{cf, newSettings}, "")
+	err = ioutil.WriteFile(LxcConfigFile, []byte(result), 0644)
+	if err != nil {
+		log.Fatalln("Couldn't save lxc config file", err)
+	}
+}
+
+func updateSettings(nwi NetworkInterface, input string) string {
+	
+	output := input + "\n"
+
+	output += fmt.Sprintf("lxc.network.veth.pair=%s\n", nwi.TapName)
+
+	if nwi.Ipv4Addr != "" {
+		output += fmt.Sprintf("lxc.network.ipv4=%s\n", nwi.Ipv4Addr)
+	}
+
+	if nwi.MacAddr != "" {
+		output += fmt.Sprintf("lxc.network.hwaddr=%s\n", nwi.MacAddr)
+	}
+
+	switch nwi.Type {
+	case "linux":
+		output += fmt.Sprintf("lxc.network.script.up=%s\n", settings.ScriptPath+settings.LinuxUpScript)
+
+		output += fmt.Sprintf("lxc.network.script.down=%s\n", settings.ScriptPath+settings.LinuxDownScript)
+
+		os.Setenv("BRIDGENAME", settings.BridgeName)
+		os.Setenv("TAPNAME", nwi.TapName)
```

Comment:
I'm using lxc-1.0.9 on CentOS7. 

```
$ rpm -qa  lxc
lxc-1.0.9-1.el7.x86_64
```
``ovs_*`` functions aren't there in v1.0.9. So please ignore my mention. 

https://github.com/lxc/lxc/blob/lxc-1.0.9/src/lxc/network.c